developer-docs-site/docs/guides/resource-accounts/key-rotation.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Key Rotation"
+id: "key-rotation"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Aptos Move accounts have authentication keys that are separate from their public address. This means you can rotate an account's private key to give control of it to another account without changing the initial account's public address.
+
+After rotating the key, the controlling account can sign transactions for the initial account.
+
+In this guide, we show examples for how to rotate an account's authentication key using a few of the various Aptos SDKs.
+
+Here are the installation links for the SDKs we will cover in this example:
+
+* [Aptos CLI](../../tools/install-cli)
+* [Typescript SDK](../../sdks/ts-sdk/index)
+* [Python SDK](../../sdks/python-sdk)
+
+:::warning
+Some of the following examples use private keys. Do not share your private keys with anyone.
+:::
+
+## How to rotate an account's authentication key
+<Tabs groupId="examples">
+  <TabItem value="CLI" label="CLI">
+
+```shell title="Initialize two test profiles on devnet"
+echo "devnet" | aptos init --profile rotate-1
+echo "devnet" | aptos init --profile rotate-2
+```
+```shell title="Rotate the authentication key for rotate-1 to rotate-2's authentication key"
+aptos account rotate-key --profile rotate-1 --new-private-key <ROTATE_2_PRIVATE_KEY>
+```
+:::info Where do I get the private key for a profile?
+Public, private, and authentication keys for Aptos CLI profiles are stored in `~/.aptos/config.yaml` if your config is set to `Global` and `<local_directory>/.aptos/config.yaml` if it's set to `Workspace`.
```

Comment:
auth key and address are a little confusing since their interchangeable until they're not